File: Java/Spring/Security_DB_Auth/src/main/java/com/github/spdeepak/security/repository/RoleRepository.java
package com.github.spdeepak.security.repository;

import com.github.spdeepak.security.entity.Role;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface RoleRepository extends JpaRepository<Role,Long> {

}

File: Kotlin/Spring/DB-Auth-Security/src/main/kotlin/com/github/spdeepak/dbauthsecurity/repository/RoleRepository.kt
package com.github.spdeepak.dbauthsecurity.repository

import com.github.spdeepak.dbauthsecurity.entity.Role
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository


@Repository
interface RoleRepository : JpaRepository<Role, Long> {

    fun findByRolename(rolename: String): Role

}